boltdesignsystem/bolt
# DS-811 Wrapper Element

## Jira

https://pegadigitalit.atlassian.net/browse/DS-811

## Summary

A new wrapper element is created.

## Details

A new wrapper is a basic element that contains a basic schema with `content` and `attributes`.

## How to test

Pull the branch. Check the wrapper docs and confirm it's a good equivalent of the old wrapper object.

## Release notes

The new `e-bolt-wrapper` will replace the old `o-bolt-wrapper` object.
